src/ToastProvider.js

```diff
@@ -62,6 +62,8 @@ type Props = {
   // A convenience prop; the duration of the toast transition, in milliseconds.
   // Note that specifying this will override any defaults set on individual children Toasts.
   transitionDuration: number,
+  // have the newest toast at the top of the stack 
+  newestOnTop: boolean
```

Comment:
Please add docs to the [README](https://github.com/jossmac/react-toast-notifications#toastprovider-props).

Reply:
Please update [types](https://github.com/jossmac/react-toast-notifications/blob/master/index.d.ts#L44).

Reply:
Please sort alphabetically, like the other props.